docs/api.rst

```diff
@@ -0,0 +1,45 @@
+API
+===
+
+.. toctree::
+   :maxdepth: 2
+
+.. _api-tracing:
+
+Tracing
+^^^^^^^
+
+.. py:module:: ddtrace
+
+.. autofunction:: ddtrace.patch_all
```

Comment:
The typing on `**patch_modules` gets shown as `dict` which isn't wrong, but interesting when it is annotated as `bool`

Reply:
Interesting! We should pay attention when reviewing docstrings to the typing hints.